pkgs/development/libraries/gtkd/default.nix

```diff
@@ -118,6 +120,13 @@ in stdenv.mkDerivation rec {
     "PKG_CONFIG=${pkgconfig}/bin/pkg-config"
   ];
 
+  postFixup = ''
+    for pc in $dev/lib/pkgconfig/*; do
+      substituteInPlace $pc \
+        --replace "$out/include" "$dev/include"
+    done
```

Comment:
Doesn't the multiple outputs setup hook do this?

Reply:
Looks like that only works on lines starting with `includedir=`, but the gtkd pc files doesn't have any variables unfortunately.

Reply:
Ah right, so it's `cflags` and `libs`.
Can you comment this? I believe in the past gtkd's .pc files have gone through some problematic issues.